public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/cron.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\{get_email_id, queue_survey_email};
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\{disable_page, publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+const TEMP_ATTENDEE_ID   = 'organizer_debrief_survey_temp_attendee';
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_organizer_debrief_survey_email', __NAMESPACE__ . '\queue_organizer_survey' );
+add_action( 'wc_organizer_disable_debrief_survey', __NAMESPACE__ . '\disable_organizer_survey' );
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
+}
+
+/**
+ * Get Wordcamp attendee ID.
+ *
+ * @return int
+ */
+function get_wordcamp_attendees_id() {
+	return get_option( TEMP_ATTENDEE_ID );
+}
+
+/**
+ * Add cron jobs to the schedule.
+ */
+function schedule_jobs() {
+	if ( ! wp_next_scheduled( 'wc_organizer_debrief_survey_email' ) ) {
+		wp_schedule_event( time(), 'daily', 'wc_organizer_debrief_survey_email' );
+	}
+}
+
+/**
+ * Add temporary attendee since email can only be sent to items in attendee tracker.
```

Comment:
This feels like a bit of a hack. I wonder if there's a way to make the CampTix email more flexible, so that it doesn't require an attendee?

Or maybe this plugin shouldn't be so tightly coupled with CampTix?

I can see how it's a practical solution, though, so it may not be worth changing. What do you think?

Reply:
> Or maybe this plugin shouldn't be so tightly coupled with CampTix?

When I was working on this PR previously, I had intended to use the Organizer Reminder to address it. However, my initial approach was in the totally wrong direction, leading me to spend an excessive amount of time on it. That's why I eventually resorted to this hacky but practical method.

Today I spent another day trying to see if the Organizer Reminder could potentially resolve this issue. I believe it's feasible, but the main hurdle is the spam concern mentioned above. If we can bypass the need to set the page to draft for spam prevention, then based on how Organizer Reminder operates, the solution would involve setting up an Automated Job with the email content prepared, including the link to the published survey page. It would then scan the WordCamp posts on a daily basis to identify any WordCamp events that meet the sending-reminder criteria, fetch their data, include it in the context of the automated job, and send the email.

What do you think of this approach?

Reply:
> I can see how it's a practical solution, though, so it may not be worth changing.

If we proceed with merging the current PR, as I've made some preparations here to generalize the plugin, we can open another PR to integrate the Attendee Survey, integrating it into a single Survey Plugin.

Reply:
That approach with Organizer Reminders sounds good 👍🏻 I also think it's reasonable to stick with this approach if you want. Organizer Reminders may be more ideal, but it also may take more time, so it's a trade off. If we'll want to refactor this later, though, then that also takes time 🤷🏻‍♂️ 

I don't have a strong opinion either way, but I trust your judgement 👍🏻

Reply:
Let's prioritize merging this over architecture changes. This feature hasn't proved its worth yet.

Reply:
Thanks for providing extra insights on the path moving forward @iandunn and @StevenDufresne, I'll proceed with the current approach.